packages/eas-cli/src/credentials/android/actions/AssignGoogleServiceAccountKeyForFcmV1.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  CommonAndroidAppCredentialsFragment,
+  GoogleServiceAccountKeyFragment,
+} from '../../../graphql/generated';
+import Log from '../../../log';
+import { CredentialsContext } from '../../context';
+import { AppLookupParams } from '../api/GraphqlClient';
+
+export class AssignGoogleServiceAccountKeyForFcmV1 {
```

Comment:
Let's refactor this into the existing [AssignGoogleServiceAccountKey](https://github.com/expo/eas-cli/blob/eb9152aa9c9590ca8720496b22c6085776217428/packages/eas-cli/src/credentials/android/actions/AssignGoogleServiceAccountKey.ts#L9). We can have a parameter for the service we want to assign the GSAK to (submissions vs FCM V1)